packages/libsolv/libsolv.spec

```diff
@@ -376,7 +385,7 @@ set -ex
 %files -n %{?scl_prefix}python%{python3_pkgversion}-%{libname}
 %{python3_sitearch}/_%{libname}.so
 %{python3_sitearch}/%{libname}.py
-%if 0%{?!scl:1}
```

Comment:
Should the whole spec file drop SCL support?

Reply:
Probably, I was going to look at scl after the release